app/main.ipynb

```diff
@@ -100,18 +119,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "length = np.random.randint(5, 10)\n",
+    "numpy_array = np.random.uniform(0, 10, size=length)\n",
+    "first_element = numpy_array[0]\n",
+    "second_element = numpy_array[2]\n",
```

Comment:
The variable `second_element` should be renamed to `third_element` to correctly reflect its position in the array.